File: Deafios em C#/1180 - Menor e Posição.cs
using System; 

//Developed by: @LucasMarcuzo

class URI1180 {

    static void Main(string[] args) { 

        int n, x, y=1000, z;
        x=z=0;

        n=int.Parse(Console.ReadLine());

        int[] vet = new int[n];

        string[] linha = Console.ReadLine().Split(' ');

        for (int i = 0; i <vet.Length; i++)
        {
            vet[i] = int.Parse(linha[i]);

            if(vet[i] < y)
            {
                y = vet[i];
                z=i;
            }
        }

        Console.WriteLine("Menor valor: {0}", y);
        Console.WriteLine("Posicao: {0}", z);
    }
}

File: Desafios em JavaScript/1180 - Menor e Posição.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var n=parseInt(lines.shift());

var ln = lines.shift().split(` `);

var vet = new Array(n);

for (let i = 0; i <vet.length; i++)
{
    vet[i] = parseInt(ln.shift());
}

var menorValor = Math.min(...vet);
var posicao = vet.indexOf(menorValor);

console.log(`Menor valor: ${menorValor}`);
console.log(`Posicao: ${posicao}`);